ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1888,12 +1888,13 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-secureboot
-- as: vsphere-upi-p3-f14
-  cron: 32 3 5,20 * *
+- as: vsphere-upi-bios-p3-f14
+  cron: 32 2 7,22 * *
   steps:
     cluster_profile: vsphere
     env:
-      E2E_RUN_TAGS: '@amd64 and @vsphere-upi and @network-openshiftsdn and not @fips'
+      E2E_RUN_TAGS: '@amd64 and @vsphere-upi and @network-ovnkubernetes and not @fips'
```

Comment:
We should revert the change on this line.  As in 4.11 and earlier, network defaults to SDN, while in 4.12 and above, the network defaults to OVN.